src/SDK/Client/Platform/methods/names/register.ts

```diff
@@ -3,19 +3,33 @@ const entropy = require('@dashevo/dpp/lib/util/entropy');
 const { hash } = require('@dashevo/dpp/lib/util/multihashDoubleSHA256');
 const bs58 = require('bs58');
 
+/**
+ * Register names to the platform
+ *
+ * @param {Platform} this - bound instance class
+ * @param {string} name - name
+ * @param identity - identity
+ * @param {any} [identity.id] - identity ID
+ * @param {number} [identity.type] - identity type
+ * @param {[any]} [identity.publicKeys] - identity public keys
+ * @param {function(number):any} - get public key by ID
+ * @returns registered names
+ */
 export async function register(this: Platform,
                                name: string,
                                identity: {
                                     id: any;
                                     type: number,
-                                   publicKeys: [any]
+                                   publicKeys: [any],
```

Comment:
nit: the indentation of this line and the next two looks different than the preceding lines